Assets/Scripts/Player/CharacterBehaviour.cs

```diff
@@ -129,10 +129,13 @@ void Update()
         // Player and Camera rotation
         if (!PauseMenu.GameIsPaused)
         {
-            rotationX += -Input.GetAxis("Mouse Y") * lookSpeed;
+            //Vertical Rotation (look up and down)
+            rotationX += -Input.GetAxis("Mouse Y") * lookSpeed * SettingsMenu.mouseSens;
```

Comment:
equivalent to `rotationX -= Input.GetAxis("Mouse Y") * lookSpeed * SettingsMenu.mouseSens;`, which is clearer to read

Reply:
If it works, it works. Lol.